### DISCUSSION TITLE
系统重启之后 AdGuard  DNS封锁清单 里面开启的DNS拦截规则，规则数为0，不手动点击更新的话拦截不起作用

### DISCUSSION THREAD

**Discussion by llivenan** [AdguardTeam/AdGuardHome#2748] [Q&A]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by Aikatsui** [comment]
1. 什麼是操作系統？
2. 提供日誌文件 [verboselog](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog)
